README.md

```diff
@@ -13,21 +13,24 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
 
 ## Quickstart
 
-Kubeapps assumes a working Kubernetes cluster (v1.8+) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and configured to talk to your Kubernetes cluster. Kubeapps binaries are available for Linux, OS X and Windows, and Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), `minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
+Kubeapps assumes a working Kubernetes cluster (v1.8+) and [`Helm`](https://helm.sh/) (2.9.1+) installed in your cluster. Kubeapps has been tested with Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), `minikube` and Docker for Desktop Kubernetes. Kubeapps works on RBAC-enabled clusters and this configuration is encouraged for a more secure install.
 
 > On GKE, you must either be an "Owner" or have the "Container Engine Admin" role in order to install Kubeapps.
 
-The simplest way to try Kubeapps is to deploy it with the Kubeapps Installer on [minikube](https://github.com/kubernetes/minikube). Assuming you are using Linux or OS X, run the following commands to download and install the Kubeapps Installer binary:
+Use the Helm chart to install the latest version of Kubeapps:
```

Comment:
Maybe add a link to the chart readme by saying something like "Click here for more information on how to configure your deployment"

Reply:
actually NM, I saw that the link is down there. I am wondering if we should move it up here.

> For information on installing and configuring Kubeapps, checkout the [chart README](chart/kubeapps/README.md).

Reply:
agreed, I moved it up